gensim/test/test_csparse.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+from gensim.csparse import gaxpy
+
+from numpy.testing import assert_array_equal
+from numpy import asfortranarray
+from numpy.random import normal
+from functools import partial
+from scipy.sparse import rand
+from os import cpu_count
```

Comment:
this doesn't works on `python2`, better use https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count